packages/@adobe/spectrum-css-temp/components/dialog/index.css

```diff
@@ -301,6 +301,28 @@ governing permissions and limitations under the License.
   }
 }
 
+/* additional a11y support by moveing scrolling from content to dialog */
+@media screen and (max-height: 400px) {
+  .spectrum-Dialog .spectrum-Dialog-grid {
+    border-top-left-radius: var(--spectrum-dialog-border-radius);
+    border-top-right-radius: var(--spectrum-dialog-border-radius);
+    overflow-y: auto;
+  }
+
+  .spectrum-Dialog-content {
+    overflow-y: visible;
+    height: min-content;
+  }
+
+  .spectrum-Dialog-footer {
+    min-width: fit-content;
```

Comment:
why fit-content? why not 0?

Reply:
I tried 0 since that worked for the ButtonGroup, but it doesn't work here. The theory I wrote you previously was `Looking at the inspection tool, it looks like this takes the footer from having almost no width (being squashed to nothing), to a width the represents the space taken by its children.`

Reply:
To see this issue, if you try to replace it inline, everything looks fine, flex won't recalculate. However, if you zoom out and back in, then you can see it do the collapsing described here.
This seems to be the most correct solution and works even if you make the checkbox label extremely long